Added RC3 detection in PF-build.sh

3d-gussner opened the pull request:
As we we have got a RC3 version I have updated also `PF-build.sh` to detect it correctly.

Test results the `PF-build.sh Version: 1.0.6-Build_14`
```
Variant    : 1_75mm_MK3S-EINSy10a-E3Dv6full
Firmware   : 390-RC3
Build #    : 3401
Dev Check  : RC3
DEV Status : RC
Motherboard: BOARD_EINSY_1_0a
Languages  : EN_ONLY
Hex-file Folder: PF-build-hex/FW390-RC3-Build3401/BOARD_EINSY_1_0a
```

3d-gussner committed:
Added RC3 detection in PF-build.sh

As we we got a RC3 version I have updated also `PF-build.sh` to detect it correctly.

deliopoulos commented:
> 
> 
> As we we have got a RC3 version I have updated also `PF-build.sh` to detect it correctly.
> 
> Test results the `PF-build.sh Version: 1.0.6-Build_14`
> 
> ```
> Variant    : 1_75mm_MK3S-EINSy10a-E3Dv6full
> Firmware   : 390-RC3
> Build #    : 3401
> Dev Check  : RC3
> DEV Status : RC
> Motherboard: BOARD_EINSY_1_0a
> Languages  : EN_ONLY
> Hex-file Folder: PF-build-hex/FW390-RC3-Build3401/BOARD_EINSY_1_0a
> ```

It's better to change the PF-build.sh line to:
`if [[ "$DEV_CHECK" == *"RC"* ]] ; then`

3d-gussner committed:
Accecpt all RCx as RC versions
Thanks to @deliopoulos

3d-gussner committed:
Adding copy of `not_tran.txt` and `not_used.txt`
to make it easier to find not translated and unused messages.
This will hopfully improve the translation process.
